crates/api_models/src/payments.rs

```diff
@@ -627,7 +627,10 @@ pub enum BankRedirectData {
         // Issuer value corresponds to the bank
         issuer: api_enums::BankNames,
     },
-    Przelewy24 {},
+    Przelewy24 {
+        // Shopper Email
+        email: Option<Email>,
```

Comment:
You can remove option for the email as earlier we don't know what could be the data needed for this payment method.

Reply:
Done👍